docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+# JupyterLab
+
+ - Even if you don't use Jupyter, you can still take this survey. Just indicate this fact in the first question and carry on as best as you can.
+ - Thank you - your participation guides Jupyter's roadmap toward your real-life use cases.
+ - So that you know what to expect, it's comprised of 20 questions (#7 is the only big one) spread across these sections:
```

Comment:
Would prefer to set expectations by directly confronting the issue with transparency rather than to blindside them. This approach has worked for me in many customer-facing scenarios. It also builds trust. I rewording it to be truer: " As a fair heads up, Question 7  is the biggest one, but it provides critical information."

Reply:
I like your suggested change 👍 Thanks @LayneSadler :sunny: